TASVideos/Pages/Games/Goals/List.cshtml

```diff
@@ -22,12 +22,15 @@
 		.ThenBy(g => g.DisplayName))
 	{
 		var canDelete = !goal.Publications.Any() && !goal.Submissions.Any();
-		<tr>
+		<tr id="@goal.Id">
 			<td condition="@(goal.Id == Model.GoalToEdit && goal.DisplayName != "baseline")">
 				<form method="post" class="d-flex gap-2" asp-page-handler="Edit">
-					<input type="hidden" name="gameGoalId" value="@Model.GoalToEdit" />
-					<input type="text" name="newGoalName" class="form-control" value="@goal.DisplayName" />
-					<button type="submit" class="btn btn-secondary">Save</button>
+					<column>
```

Comment:
you don't want to use column without also using a column for row.  you can use <full-row> which will add both of those for you